modules/planning/common/obstacle.h

```diff
@@ -124,6 +124,10 @@ class Obstacle {
 
   static bool IsValidTrajectoryPoint(const common::TrajectoryPoint& point);
 
+  const bool IsCautionLevelObstacle() const {
```

Comment:
1. Don't return a `const` copy which has no benefit but harm.
2. Better to declare `inline` for functions in header file, or you may run into "redefinition" error.

Reply:
Fixed. Thanks!